content/docs/introduction/faq.md

```diff
@@ -117,11 +117,15 @@ for the current state of this effort.
 
 ### Which languages have instrumentation libraries?
 
-Currently, there are client libraries for:
+Currently, there are official client libraries for:
 
 * [Go](https://github.com/prometheus/client_golang)
-* [Java or Scala](https://github.com/prometheus/client_java)
+* [Java (JVM)](https://github.com/prometheus/client_java)
 * [Ruby](https://github.com/prometheus/client_ruby)
+* [Python](https://github.com/prometheus/client_python)
```

Comment:
Is it important to list the client libraries here directly? Having more than one place where something is documented will easily lead to inconsistencies again. What about only linking to the clientlibs page here, so that this kind of problem can't happen again? @juliusv